addons/l10n_ec/data/account_tax_report_data.xml

```diff
@@ -88,6 +88,42 @@
                             </record>
                         </field>
                     </record>
+                    <record id="tax_report_line_parent_line_report_7" model="account.report.line">
+                        <field name="name">Local sales (excluding fixed assets) taxed at 5%</field>
+                        <field name="children_ids">
+                            <record id="tax_report_line_104_425" model="account.report.line">
+                                <field name="name">Gross value(425)</field>
+                                <field name="code">c425_104</field>
```

Comment:
Let's keep because:
1. To have all taxes with the same structure (all taxes have a code)
2. To in the far future improve the tax report, adding up to subtotals (this tax should be, one day, added up into other tags with subtotals)
3. The code usually would have been just "c425" but is colliding with a code with same name used for another type of tax report (this is the vat tax report usually called the 104 report, collides with another code from profit withholds tax report usally called the 103 report), so we had to gave it a slightly different name

Reply:
@pepetreshere Normally the code has to be unique per report (looking in https://github.com/odoo/odoo/blob/0d5bf820c3737ee3e4af54d1fb556b72d6c59c3d/addons/account/models/account_report.py#L326 )